src/js/components/SingleFood.scss

```diff
@@ -0,0 +1,13 @@
+// sf means single food
```

Comment:
Just a small note - if it means single food, you might as well call it `.single-food`. No need to use shorter names. :)

Reply:
haha. Jamie rubbed off on me. I'll change moving forward.